PendingReleaseNotes

```diff
@@ -198,6 +198,8 @@
   also change read/write permissions in a capability that the entity already
   holds. If the capability passed by user is same as one of the capabilities
   that the entity already holds, idempotency is maintained.
+* `ceph config dump --format <json|xml>` will now show the localized option
```

Comment:
nit: _shows now_?

Reply:
@rzarzynski I modified the note to include bit more detail. PTAL.